src/node/blockstorage.h

```diff
@@ -429,7 +430,7 @@ class BlockManager
     void CleanupBlockRevFiles() const;
 };
 
-void ImportBlocks(ChainstateManager& chainman, std::vector<fs::path> vImportFiles);
+void ImportBlocks(ChainstateManager& chainman, std::span<const fs::path> vImportFiles);
```

Comment:
nit: would be nice to modernize the name to e.g. `import_paths` since it's no longer a vector.
```suggestion
void ImportBlocks(ChainstateManager& chainman, std::span<const fs::path> import_paths);
```

Reply:
Oh right, finally!